manifests/profile/server.pp

```diff
@@ -375,7 +380,7 @@
     }
 
     St2::Package::Install<| tag == 'st2::profile::server' |>
-    -> Ini_setting<| tag == 'st2::profile::server' |>
+    -> Ini_setting<| tag == 'st2::config' |>
     ~> Service<| tag == 'st2::profile::server' |>
```

Comment:
@Kami this is the service refresh. Now, any `Ini_setting` with this tag will also restart.